### PR TITLE
Depreciating Push registration endpoint

### LINKED ISSUE #173: Remove the Push Registration Endpoint
With the incorporation of the Push blade into Azure App Service, we need to deprecate the Push Registration endpoint.

### PR DESCRIPTION
**Pull request by pvbchak**
Fixes #173

### CODE REVIEW

File: src/Microsoft.Azure.Mobile.Server.Authentication/Microsoft.Azure.Mobile.Server.Authentication.nuspec [hunk 1]
```diff
@@ -11,7 +11,10 @@
     <projectUrl>https://github.com/azure/azure-mobile-apps-net-server</projectUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <description>This package is part of the Azure Mobile .NET Server SDK and provides authentication features. To learn more about Azure Mobile, go to http://azure.microsoft.com/mobile.</description>
-    <releaseNotes></releaseNotes>
+    <releaseNotes>
+      Breaking Changes
+        Version 2.0.0 : ProviderCredentials.Claims is no longer available. User ProviderCredentials.UserClaims instead
```
Comment (adrianhall) [inline src/Microsoft.Azure.Mobile.Server.Authentication/Microsoft.Azure.Mobile.Server.Authentication.nuspec]: nit: "User ProviderClaims.UserClaims" => "Use ProviderClaims.UserClaims"

File: src/Microsoft.Azure.Mobile.Server.Notifications/Microsoft.Azure.Mobile.Server.Notifications.nuspec [hunk 2]
```diff
@@ -3,15 +3,18 @@
   <metadata>
     <id>Microsoft.Azure.Mobile.Server.Notifications</id>
     <version>$version$$prerelease$</version>
-    <title>Azure Mobile .NET Server Notifications</title>
+    <title>(DEPRECIATED) Azure Mobile .NET Server Notifications</title>
```
Comment (adrianhall) [inline src/Microsoft.Azure.Mobile.Server.Notifications/Microsoft.Azure.Mobile.Server.Notifications.nuspec]: Nit: Spelling mistake.

File: src/Microsoft.Azure.Mobile.Server.Notifications/Microsoft.Azure.Mobile.Server.Notifications.nuspec [hunk 3]
```diff
@@ -3,15 +3,18 @@
...
     <owners>Microsoft azure-sdk</owners>
     <licenseUrl>http://go.microsoft.com/fwlink/?LinkID=331327</licenseUrl>    
     <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288890</iconUrl>
     <projectUrl>https://github.com/azure/azure-mobile-apps-net-server</projectUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <description>This package is part of the Azure Mobile .NET Server SDK. Use it to register push notifications with Azure Notification Hubs. To learn more about Azure Mobile, go to http://azure.microsoft.com/mobile.</description>
-    <releaseNotes></releaseNotes>
+    <releaseNotes>
+      Breaking Changes
+        Version 2.0.0 : This package is depreciated &amp; will not receive major updates. Please use App Service Push instead.
```
Comment (adrianhall) [inline src/Microsoft.Azure.Mobile.Server.Notifications/Microsoft.Azure.Mobile.Server.Notifications.nuspec]: Nit: deprciated => deprecated

**Comment by dhei** [comment]
looks good to me

File: src/Microsoft.Azure.Mobile.Server.Notifications/Microsoft.Azure.Mobile.Server.Notifications.nuspec (hunk 3 above)
Comment (pvbchak) [inline src/Microsoft.Azure.Mobile.Server.Notifications/Microsoft.Azure.Mobile.Server.Notifications.nuspec]: fixed.

File: test/Microsoft.Azure.Mobile.Server.Notifications.Test/Extensions/MobileAppExtensionsTests.cs [hunk 4]
```diff
@@ -43,12 +43,12 @@ public class MobileAppExtensionsTests
                     .Returns(Task.FromResult(0));
 
                 config.SetPushClient(pushClientMock.Object);
-
+#pragma warning disable 618
```
Comment (pvbchak) [inline test/Microsoft.Azure.Mobile.Server.Notifications.Test/Extensions/MobileAppExtensionsTests.cs]: Fixed

File: src/Microsoft.Azure.Mobile.Server.Authentication/Microsoft.Azure.Mobile.Server.Authentication.nuspec (hunk 1 above)
Comment (pvbchak) [inline src/Microsoft.Azure.Mobile.Server.Authentication/Microsoft.Azure.Mobile.Server.Authentication.nuspec]: Fixed.

File: src/Microsoft.Azure.Mobile.Server.Notifications/Microsoft.Azure.Mobile.Server.Notifications.nuspec (hunk 2 above)
Comment (pvbchak) [inline src/Microsoft.Azure.Mobile.Server.Notifications/Microsoft.Azure.Mobile.Server.Notifications.nuspec]: Fixed.

**Pull request merged**